cibuildwheel/platforms/android.py

```diff
@@ -508,17 +508,21 @@ def repair_default(
         new_soname = soname_with_hash(src_path)
         dst_path = libs_dir / new_soname
         shutil.copyfile(src_path, dst_path)
-        call("patchelf", "--set-soname", new_soname, dst_path)
+
+        # If cibuildwheel was called without activating its environment, the `bin`
+        # directory will not be on the PATH.
+        cibw_bin = Path(sys.executable).parent
```

Comment:
This is not guaranteed to be present. On conda-forge, patchelf comes from the feedstock, not our install. So I think this should revert (or use `shutil.which`) if `cibw_bin / "patchelf"` doesn't exist.

Reply:
Thanks, I've gone with Joe's suggestion.